WhatIsASprint.md

```diff
@@ -0,0 +1,23 @@
+
+# **What is a Sprint?**
+
+
+A **_sprint_** is a set period of time broken out from a project's overall timeline with associated tasks to be completed during that period of time. That list of tasks is referred to as the sprint's <b>Product Backlog</b>.
+
+
+A *sprint* allows a larger timeline to be easily broken into smaller, more digestible pieces in order to focus effort from all stakeholders on a few logically scheduled tasks at a time.
```

Comment:
As you yourself have described earlier, _sprint_ is the slice of a larger timeline. That means this sentence cannot be true. Because _sprint_ is **not** the process of breaking it down.

What allows for a larger timeline to be broken into pieces is human decision making, and sprint is the result of that.